app/models/repo_gemnasium_data.rb

```diff
@@ -12,12 +12,32 @@ def self.run
   end
 
   def load_data
+    load_alerts
+    load_dependencies
+    true
+  end
+
+  private
+
+  def load_alerts
     client.all_projects_and_counts.each do |slug, advisories|
       organization, name = slug.split('/')
       Repository.find_by(name: name, organization: organization).tap do |repo|
         repo&.update(gemnasium_alerts: advisories)
       end
     end
-    true
+  end
+
+  def load_dependencies
+    client.all_dependencies.each do |slug, dependencies|
+      deps = []
+      dependencies.each do |d|
+        deps.push(Dependency.new(name: d['name'], version: d['version']))
+      end
```

Comment:
This could be done with:
```
deps = dependencies.map do |d|
  Dependency.new(name: d['name'], version: d['version'])
end
```
as `.map` will always return and array